### PR TITLE
HARVESTER: harvester bug fix

### PR DESCRIPTION
**Pull request by WuJun2016**
# link
rancher:
- https://github.com/rancher/dashboard/issues/4299

Fix isNamespaced's resource filter condition

harvester:
singleVirtualCluster bug fix

### CODE REVIEW

File: plugins/extend-router.js [hunk 1]
```diff
@@ -19,6 +19,43 @@ VueRouter.prototype.applyQuery = function(qp, defaults = {}) {
   });
 };
 
+// https://github.com/vuejs/vue-router/issues/2881#issuecomment-520554378
```
Comment (nwmac) [inline plugins/extend-router.js:22]: Do we need this? Just concerned about introducing this plugin at this stage - we should probably just add catches to the router.push/replace

**Comment by WuJun2016** [comment]
@nwmac   Remove the code for the relevant routing changes.   Could you review it again. thanks

File: components/YamlEditor.vue [hunk 2]
```diff
@@ -142,6 +142,14 @@ export default {
       if (neu) {
         this.$refs.yamluploader.click();
       }
+    },
+
+    value(neu) {
```
Comment (nwmac) [inline components/YamlEditor.vue:147]: Are you sure this is needed? We had issues with putting this in in the YamlEditor as it caused a loop of change events across the 2-way binding

**Review by nwmac — COMMENTED** [review]
One comment to look at

**Comment by WuJun2016** [comment]
Done. Thanks for your review.  I submitted some new commits to fix some issues.  Could you review it again. thanks.

**Review by nwmac — APPROVED** [review]
Approved - this is marked asa DRAFT PR - when you're ready for it to be merged, can you convert it to a regular PR

**Pull request merged**